terraform-post-packer/main.tf

```diff
@@ -29,7 +29,7 @@ data "aws_ami_ids" "historical_amis" {
   filter {
     name = "name"
     values = [
-      "example-hvm-*-x86_64-ebs",
+      "windows-server-2022-*-x86_64-ebs",
```

Comment:
Do we want to update the AMI name?

Reply:
This change was unintentional. Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/20/commits/fca2728ac8fdb8298991237ff2c1d66dc921be17#diff-61505b29b2bfc888d227eb40217a3c6d44ec3ba4dc88b44bdf89a7fa63d3358fR88

Reply:
Addressed here https://github.com/cisagov/windows-commando-vm-packer/commit/fca2728ac8fdb8298991237ff2c1d66dc921be17#diff-686c4544bf4e69fe6ff1cfa9640a1d3bf44d8f775eaf9c9038000ed382ad2dd0R32